rdd/airports/AirportsByLatitudeSolution.py

```diff
@@ -0,0 +1,17 @@
+from pyspark import SparkContext
+from commons.Utils import Utils
+
+def splitComma(line: str):
```

Comment:
Have you tried to run this program? It doesn't compile
```
  File "/Users/cwei/code/python-spark-tutorial-new/rdd/airports/AirportsByLatitudeSolution.py", line 4
    def splitComma(line: str):
                       ^
SyntaxError: invalid syntax
```

Reply:
Hi, yes I did ran all programs. Which version of python are you running? This should work in the latest Python 3 version

Reply:
sorry for the confusion. It works for Python 3. I was running python 2.7. Feel free to ignore this comment.